app/management/commands/rundev.py

```diff
@@ -2,6 +2,63 @@
 import subprocess
 from django.core.management.base import BaseCommand
 
+def check_env():
+    # checks .env for all the required fields
+    # print('Checking Environment Variables...')
+    file_dir = os.path.dirname(os.path.realpath('__file__'))
+    try:
+        env_file = open(os.path.join(file_dir,'.env'))
+    except FileNotFoundError as exc:
+        raise FileNotFoundError(
+                "The .env file does not exist, you should create it!"
+                ) from exc
+        exit()
+    try:
+        env_check_file = open(os.path.join(file_dir,'env_check_file'))
+    except FileNotFoundError:
+        print('env_check_file not found, creating...')
+        env_check_file = open(os.path.join(file_dir,'env_check_file'), 'x')
+        env_check_file = open(os.path.join(file_dir,'env_check_file'))
+    counter = 0
+    for check_line in env_check_file:
+        counter += 1
+        LINE_EXISTS = False
+        check_line = check_line.replace("\n","")
+        check_line = check_line.replace(" ","")
+        if check_line == "":
+            continue
+        env_file_lines = open(os.path.join(file_dir,'.env'))
+        for env_line in env_file_lines:
+            if env_line.replace(" ","").replace("\n","") == "":
+                continue
+            if check_line in env_line.partition("=")[0]:
+                LINE_EXISTS = True
+                break
+        if LINE_EXISTS == False:
+            print(f".env file missing variable {check_line}")
+
+def check_env_updater():
+    # automatically updates env_check_file with all the variables from .env
+    # print('Updating env_check_file...')
+    file_dir = os.path.dirname(os.path.realpath('__file__'))
+    env_file = open(os.path.join(file_dir,'.env'))
+    env_check_file = open(os.path.join(file_dir,'env_check_file'))
```

Comment:
if your opening a file like this make sure you close it or use `with open` to automatically do so